### DISCUSSION TITLE
CLI DID export with no private key

### DISCUSSION THREAD

**Discussion by manicprogrammer** [decentralized-identity/veramo#1303] [General]
Using v5.5.2 command line there is a `veramo did export` command but I am not sure what value it provides as I find no private key in the export file and thus not sure what it could be imported into. Am I alone that in the export I should expect a private key?

**Comment by mirceanis** [comment]
Good catch.
You are right to expect a private key there.

I think the did export command should be removed as it was added when private keys were also exportable, which is not the case anymore, or at least it's not the case with all key management systems.
